src/main/vue/src/store/sound/mutations.ts

```diff
@@ -0,0 +1,14 @@
+import { MutationTree } from 'vuex';
+import { SoundState, Sound } from './types';
+
+export const mutations: MutationTree<SoundState> = {
```

Comment:
This pattern with separate model, mutations and getters looks very clear at first glance :+1: